src/MolSim.cpp

```diff
@@ -92,22 +101,33 @@ int main(int argc, char *argsv[]) {
     case 'x':
       xyz_output = true;
       break;
+    case 'l':
+      log_level = std::string(optarg);
+      break;
+    case 'f':
+      calculateLJForce = false;
+      break;
+    case 'n':
+      enableOutput = false;
+      break;
     default:
       fprintf(stderr, "Usage: %s [-h] help\n", argsv[0]);
       return 1;
     }
   }
 
+  Logger &logger = Logger::getInstance(log_level);
+
   FileReader fileReader;
   fileReader.readFile(particles, input_path.data());
 
   int iteration = 0;
   double current_time = start_time;
 
-  std::cout << "Starting a simulation with:\n"
-            << "\tStart time: " << start_time << "\n"
-            << "\tEnd time: " << end_time << "\n"
-            << "\tDelta: " << delta_t << "\n";
+  logger.warn("Starting a simulation with:");
```

Comment:
```suggestion
  logger.info("Starting a simulation with:");
```
I don't really get why this would be `warn` not `info`.